repositories/src/main/scala/cromwell/pipeline/datastorage/dao/repository/DocumentRepository.scala

```diff
@@ -16,5 +17,8 @@ class DocumentRepository(collection: MongoCollection[Document]) {
     collection.replaceOne(equal(fieldName, name), document, ReplaceOptions().upsert(true)).toFuture()
 
   def getByParam(field: String, name: String): Future[Seq[Document]] =
-    collection.find(regex(field, name)).toFuture()
+    collection.find(equal(field, name)).toFuture()
+
+  def deactivateOne(fieldName: String, name: String): Future[UpdateResult] =
+    collection.updateOne(equal(fieldName, name), set("active", "false")).toFuture()
```

Comment:
I'm not sure that the repository must know about the name of the field that we use as an activity field.
Maybe you should move all of this logic into `Service` layer. Read the Configuration, set activity to `false`, and then use `updateOne` method. This approach will also allow you to check that we are not trying to delete the configuration that is already removed.


Anyway, I assume activity is a boolean, not a string.
```suggestion
    collection.updateOne(equal(fieldName, name), set("active", false)).toFuture()
```

Reply:
Done

Reply:
What about moving this logic to a `Service` layer or something like that?

Reply:
Agree, already fixed